ejemplo.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>The Cheese Guardian</title>
+  <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+  <section class="main-container">
+    <img src="phone.png" alt="foto"  />
+    <div class="vertical-separator"> </div>
+    <img src="tcg_logo.png" alt="foto"  />		
+  </section>
+  <div class="mountain-image"></div>
+  <div class="second-container"> 	
+    <div class="element">
+      <img src="ic_list.png" alt="foto"  />
+      <p class="font">Keep track of your money</p>
```

Comment:
remember to use html elements related to what the text represents, this should be a title so you should use an h1, h2, h3, h4, h5, h6 element instead of a p.